app/export/formats/orchestras_item_summary_format.rb

```diff
@@ -242,5 +333,9 @@ def total_ticket_str(total_field)
     def total_instrument_str(total_field)
       "Inget: #{total_field[:none]}, Väldigt litet: #{total_field[:very_small]}, Litet: #{total_field[:small]}, Mellan: #{total_field[:medium]}, Stort: #{total_field[:large]}"
     end
+
+    def total_tshirt_str(total_field)
```

Comment:
Would it perhaps be a better idea to split this into separate columns in the spreadsheet instead of having this gigantic string?